make-compiler

```diff
@@ -5,8 +5,9 @@
 
 set -euo pipefail
 
+silver_version=`git describe --tags`
```

Comment:
```suggestion
silver_version=`git describe --tags`
echo "Build version $silver_version"

```

Maybe print the computed version  here, would be helpful for diagnosing builds on Jenkins.